source/to_cpp1.h

```diff
@@ -1497,6 +1497,13 @@ class cppfront
         //
         if (!source.has_cpp2()) {
             assert(ret.cpp2_lines == 0);
+            if (
+                cpp1_filename.back() == 'h' 
+                && flag_import_std
```

Comment:
`(flag_import_std || flag_include_std)` to match https://github.com/hsutter/cppfront/blob/68b716abe3e4b623e875d68a2b9b3f0efcd39b45/source/to_cpp1.h#L1301-L1305

Reply:
I force-pushed with your suggestion. Thanks 😊